helm/charts/hpe-csi-driver/values.yaml

```diff
@@ -23,3 +23,6 @@ logLevel: "info"
 
 # Registry prefix for hpe-csi-driver images
 registry: "quay.io"
+
+# NodeGetVolumestats will be called by default, set true to disable the call 
+disableNodeGetVolumestats: false
```

Comment:
Please update the values.yaml under values/csi-driver/v2.0.0

Reply:
done